manifests/resource/location.pp

```diff
@@ -407,8 +407,12 @@
 
   $config_file = "${server_dir}/${server_sanitized}.conf"
 
-  $location_sanitized_tmp = regsubst($location, '\/', '_', 'G')
-  $location_sanitized = regsubst($location_sanitized_tmp, '\\\\', '_', 'G')
+  if !($location =~ /^~/) {
+    $location_sanitized_tmp = regsubst($location, '\/', '_', 'G')
+    $location_sanitized = regsubst($location_sanitized_tmp, '\\\\', '_', 'G')
+  } else {
+    $location_sanitized = $location_sanitized_tmp
```

Comment:
I think you are correct. location_sanitized does not seem to be in use anymore.  See https://github.com/voxpupuli/puppet-nginx/commit/15dfc77709f6cd6a1fe4c64504a904c751df2165 for the change that removed the use of that variable.  Now it just takes an md5 of $location to build up the title of the fragment.

I'll modify this pull request to remove those lines.

You are also correct that I'm actually on 0.2.7 (bad me for trying to be sneaky)